### PR TITLE
Hello World Project

### PR DESCRIPTION
**Pull request by nmeziany**
In this Project I:
- Set up linters for HTML and CSS.
- I created an HTML file to represent the project.
- I style it using a CSS file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Not available yet
+
+
+## Authors
+
+👤 Noureddine MEZIANY
+
+- GitHub: [@nmeziany](https://github.com/nmeziany)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/noureddine-meziany/)
```
Comment (OLIPLICHE) [inline README.md]: You have done amazing work✔️. Please use the suggested ReadMe [template](https://github.com/microverseinc/readme-template) to add what is missing in your Readme ( Contributing, Show your support, Acknowledgments, License).

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi, @nmeziany👋,

Good job so far!
There are some issues that you still need to work on to get your project approved, but you are almost there!

{EC02D738-69D3-4169-B23A-40DAFCDC3CE2}


## Highlights
Excellent Job 👏 👏👏
- Linter is set up Correctly ✔️ 
- Your Linter result is green ✔️ 
- Github flow is correctly used ✔️ 
- Your commit messages are meaningful✔️
- Pull request has a descriptive summary✔️

## Required Changes ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (nmeziany) [reply README.md]: Changes done! 💯

**Review by prolajumokeoni — APPROVED** [review]
Hi @nmeziany, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**